### DISCUSSION TITLE
Is is possible to put fabric to accept input in other languages that are not english?

### DISCUSSION THREAD

**Discussion by 19jmrs** [danielmiessler/Fabric#762] [General]
I'm trying to use a transcript in portuguese with the "extract_wisdom" pattern, but it does not seem to be working, am I missing something or is it really only available in English?

**Comment by RoeslerM** [comment]
the extract_wisdom pattern itself works with any kind of language. You can validate that by simply pasting a portuguese text and pipe it to fabric -p extract_wisdom. You will get an english summary of your portuguese text, the quotes within the response will remain portuguese.

so to get this to work, you need to tell YT what transscript language you want, the default language is English.
In your case try

**`yt --lang pt --transcript [_YOUR-VIDEO-URL_]`**

this should generate a portuguese text output. If that is working, you can pipe the output to extract_wisdom pattern.

**`yt --lang pt --transcript [_YOUR-VIDEO-URL_] | fabric -sp extract_wisdom`**

**Comment by vardaasen** [comment]
I came across the same question prompting fabric

for the solution suggested above by  @RoeslerM  you need some way to enable subtitles  

A possible solution for fabric since it now also added google gemini api, and because gemini seem to accept "foreign language" in youtube videos before extracting transcripts and summarizing content, there are more ways to look at this. Using the gemini api is not free, while the youtube api is free. 

An updated version of `yt --transcript`, or some way of prompting gemini directly with a special foreing language youtube video prompt, a bit messy
